### ISSUE TITLE
Warnings on fresh install - access array offset on value of type bool in controllers/admin/Settings.php on line 254

### ISSUE THREAD

**Issue by bwmarkle** [SproutApps/sprout-invoices#32]
On a new WordPress, getting warnings:

![image](https://user-images.githubusercontent.com/5925563/98279753-5b138d80-1f68-11eb-8e41-5a359fbdae9a.png)

```
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 254
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 255
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 256
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 257
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 258
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 259
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 260
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 254
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 255
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 256
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 257
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 258
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 259
[05-Nov-2020 18:04:15 UTC] PHP Notice:  Trying to access array offset on value of type bool in /home/user/public_html/wp-content/plugins/sprout-invoices/controllers/admin/Settings.php on line 260
```

**Comment by dancameron** [comment]
Fixed in https://github.com/BoldGrid/sprout-invoices/commit/8b291e69b981c2ed6a54993a24045a256724cc1a

**Comment by elanasparkle** [comment]
Resolved with 19.9.3.5
